lib/generators/rolemodel/saas/blazer/blazer_generator.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../../../bundler_helpers'
+
+module Rolemodel
+  module Saas
```

Comment:
Saas?

Reply:
Ok. I see that's nesting approach. Could see it simply being rolemodel:blazer

Reply:
ok, will change